src/main/java/edu/wisc/my/messages/model/Message.java

```diff
@@ -322,14 +323,29 @@ public boolean isValidToday() {
 
     try {
       // if the message is premature, it is not valid.
-      if (StringUtils.isNotBlank(goLiveDate) && LocalDateTime.parse(goLiveDate).isAfter(now)) {
+      // go live date might be time specified
+      if (StringUtils.isNotBlank(goLiveDate) && goLiveDate.contains("T")
```

Comment:
This is probably the most enjoyable commit I've written in months. I really got a kick out of this. Also we shouldn't merge this until refactored -- `Message` now knows way too much nitty gritty about date parsing.